### ISSUE TITLE
Creating bit-identical docx files 

### ISSUE THREAD

**Issue by Matthias1975** [python-openxml/python-docx#1042]
Hi,

when saving docx with python-docx, the individual files in the zip file have the current date and time as the modification date.

Re-creating the files with the same input data therefore leads to non bit-identical results.

If one manages docx in version control systems, such as SVN or git, files are marked as changed, although they are identical in content.

The cause lies in the module "phys_pkg.py", class "_ZipPkgWriter", method "write". Is it possible to change the method as follows?

old:
```
def write(self, pack_uri, blob):
    self._zipf.writestr(pack_uri.membername, blob)
```

new:
```
def write(self, pack_uri, blob):
    zinfo = ZipInfo(filename=pack_uri.membername, date_time=[1980, 1, 1, 0, 0, 0])
    zinfo.compress_type=ZIP_DEFLATED

    self._zipf.writestr(zinfo, blob)
```

Btw, if a docx is saved with Word (Office365), the date is also always 1980-1-1.

Thanks a lot
Matthias

**Comment by AltayAkkus** [comment]
If I understand right:
You have a file, test.docx, that has the MD5 of XYZ.
You open it with python-docx, and close it without changing anything, and then the hash is not XYZ anymore?
And thats because python-docx alters the modified date by default?

Can you give a code example to reproduce?

**Comment by CanIGetaPR** [comment]
> If I understand right: You have a file, test.docx, that has the MD5 of XYZ. You open it with python-docx, and close it without changing anything, and then the hash is not XYZ anymore? And thats because python-docx alters the modified date by default?

No, python-docx stores a creation date in the file thus an identical document has a different MD5. Creation date has no purpose inside the file. File systems store file metadata such as creation date.

@Matthias1975 there is an active fork of docx maybe check out and report this problem on https://github.com/HiTalentAlgorithms/python-docx

**Comment by scanny** [comment]
@Matthias1975 I'm open in principle to this, in particular, matching Word behavior on the Zip member datetime seems like the right thing to do in any case.

I'm not clear about a few things:

1. Why the ZIP_DEFLATED? And what consequences are there for that choice, in particular regarding file size and save and load time?
2. What is the actual use case for this? If a file is not modified, then why would you save it? Is this just so a client program could call `document.save()` without the overhead of maintaining a dirty flag?
3. Have you actually tried it with these settings and confirmed that it worked? I'm inclined to think there are other non-deterministic factors in there that might be harder to root out. Binary-identical is a pretty unforgiving standard to meet :)
